docs/make.jl

```diff
@@ -0,0 +1,23 @@
+using SciMLModelCoupling
+using Documenter
+
+DocMeta.setdocmeta!(SciMLModelCoupling, :DocTestSetup, :(using SciMLModelCoupling); recursive=true)
+
+makedocs(;
+    modules=[SciMLModelCoupling],
+    authors="Pablo Rodríguez Sánchez <pablo.rodriguez.sanchez@gmail.com> and contributors",
+    sitename="SciMLModelCoupling.jl",
+    format=Documenter.HTML(;
+        canonical="https://pabrod.github.io/SciMLModelCoupling.jl",
+        edit_link="main",
+        assets=String[],
+    ),
+    pages=[
+        "Home" => "index.md",
+    ],
+)
+
+deploydocs(;
+    repo="github.com/pabrod/SciMLModelCoupling.jl",
```

Comment:
The link of this repo is under your name `pabrod`, shouldn't it point to the organization version? Also, the one used here does not exist.
Or do we have to change the name of the repo first?
```suggestion
    repo="github.com/DEEPDIP-project/SciML-model-coupling",
```

Reply:
True. Said this, please keep in mind that:

- Generating the website is still work in progress
- I'll do it in an independent branch/pull request